src/main/resources/creatures.json

```diff
@@ -311,7 +311,7 @@
       "tags": [
         "CORPSE"
       ],
-      "drop": [
+      "drops": [
```

Comment:
It was, but the point is that now that I've changed the rules to fail if there are fields which we did not test, this broke the build for me and showed that in a single item I wrote "drop" instead of "drops" and this caused a silent glitch. If you hadn't written all other rules everything would have failed the build, my point is that this typo would go unnoticed forever.